libs/full/resiliency_distributed/include/hpx/resiliency_distributed/async_replay_distributed.hpp

```diff
@@ -0,0 +1,184 @@
+//  Copyright (c) 2019-2020 Nikunj Gupta
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <hpx/resiliency/resiliency_cpos.hpp>
```

Comment:
The headers, tests, and examples in the `resiliency_distributed` module would still need to be wrapped in
```
#include <hpx/config.hpp>
#if !defined(HPX_COMPUTE_DEVICE_CODE)
...
#endif
```
in order for all of this to compile correctly with `hipcc`.